content/doc/developer/publishing/plugin-site.adoc

```diff
@@ -1,8 +1,98 @@
+rather explains how it gets its data and how plugin developers can
+change the data on it.
+
+== Data shown on the plugins site
+
+=== Which plugins are listed? How can I make my plugin appear?
+
+All plugins published to the current (weekly) update center are listed.
+
+=== What is the plugin installation trend based on?
+
+This uses the monthly data from http://stats.jenkins.io/ as gathered
+from Jenkins instances that have not opted out of reporting data.
+
+Learn more on the link:../usage-statistics[usage statistics page].
+
+=== What does "most installed" mean?
+
+This ranks plugins by the number of different Jenkins instances that
+reported the plugin installed during the previous month.
+
+=== What does 'Recently updated' mean?
+
+This is based on metadata created by the update site generator, which
+uses the creation date of a plugin release.
+
+Note that very recent plugin releases may not appear here due to caching
+and delays in data aggregation.
+
+=== What does 'Trending' mean?
+
+This is an attempt to capture the plugins whose popularity is growing
+rapidly. As we only have the monthly installation stats for this, it's
+necessarily a worse approximation than what you'd likely see e.g. in app
+stores.
+
+Currently (2019-10) this just divides the previous month's install count
+by the month before that. Plugins with the greatest numbers are shown in
+descending order.
+
+=== What are categories based on?
+
+Categories are based on link:https://github.com/jenkins-infra/update-center2/blob/master/src/main/resources/label-definitions.properties[label-definitions.properties]
+ in the link:https://github.com/jenkins-infra/update-center2/#categorizing-plugins[update-center2] GitHub repository.
+
+=== Where does the description of the plugin come from?
+
+The plugin site will pull from either the plugin's README page on GitHub or it's wiki page.
+
+==== GitHub
+See link:../documentation/#using-github-as-a-source-of-documentation[documentation] for 
+how to configure GitHub documentation.
+
+==== Wiki
+The plugins site scrapes the plugin's wiki page, transforms it a bit
+(e.g. makes table of content links not link to wiki.jenkins-ci.org, but
+to the plugin site), and caches it.
+
+Changes to plugin wiki pages will therefore take effect with some delay
+on the plugin site.
+
+=== What is the maintainer information based on?
+
+These are taken from the plugin's latest released version's metadata,
+and additionally (if not redundant) includes the user who released that
+version.
+
+=== What's the data source for Version, Dependencies and Display Name?
+
+These are all taken from the plugin's latest released version's pom.xml
+metadata.
+
+Implicit dependencies (plugins split from core) are included with `(Implied)` after the plugin name.
+
+== Data collection and caching
+
+=== Wiki page data
+
+The plugin site caches wiki pages as needed. The cached copy of each
+wiki page is marked invalid after several (2019-10: six) hours.
+
```

Comment:
I wonder what the duration is for GitHub.

Reply:
It seems to be the same: https://github.com/zbynek/plugin-site-api/blob/master/src/main/java/io/jenkins/plugins/services/impl/HttpClientWikiService.java#L56

but I didn't think it actually updated, i.e. I thought it was just at release time? Haven't read the code enough to see how it fully works though